Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/02-state-storage.md

```diff
@@ -1,41 +1,48 @@
 # Use the Azure CLI to create a storage account to store the Terraform state files.
 This storage account will be used to store the state of each deployment step and will be accessed by Terraform to reference values stored in the various deployment state files.
 
-Create some variables to start with
-
-```bash
-REGION=<REGION>
-STORAGEACCOUNTNAME=<UNIQUENAME>
-CONTAINERNAME=akscs
-TFSTATE_RG=tfstate
-```
-
-or with Powershell
-```powershell
-$REGION=<REGION>
-$STORAGEACCOUNTNAME=<UNIQUENAME>
-$CONTAINERNAME=akscs
-$TFSTATE_RG=tfstate
-```
-
-Create a Resource Group:
-```
-az group create --name $TFSTATE_RG --location $REGION
-```
-
-Create a Storage Account:
-```
-az storage account create -n $STORAGEACCOUNTNAME -g $TFSTATE_RG -l $REGION --sku Standard_LRS
-```
-
-Create a Storage Container within the Storage Account:
-
-```
-az storage container-rm create --storage-account $STORAGEACCOUNTNAME --name $CONTAINERNAME
-```
+1. Login to the Azure subscription that you'll be deploying into with your credentials.
+
+   ```Shell
```

Comment:
Let's just go with PowerShell then for all of these, now that we've switched.  :)  That'll make for better variable syntax highlighting.

~~~suggestion
   ```PowerShell
~~~